Automattic/wp-calypso
# Staging sites: Hide 'Accept gift subscription' and 'Site tools' on General Settings

- closes https://github.com/Automattic/dotcom-forge/issues/1902

## Proposed Changes

* Hide 'Accept gift subscription' and 'Site tools' for Staging sites on General Settings

## Testing Instructions



* Create a staging site from an atomic business site using the Hosting Configuration > Create a Staging site
* Click `Manage staging site`
* Click on Settings to navigate to general settings
* Observe the 'Accept gift subscription' and 'Site tools' cards are not visible

## Screencast

https://user-images.githubusercontent.com/779993/223762088-d78347ae-e342-470e-9d94-77c33c446610.mp4

## Screenshots

| **Before** | **After** |
|---|---|
| <img width="1278" alt="staging-sites-general-settings-before" src="https://user-images.githubusercontent.com/779993/223762331-7b3f90e6-5ba6-4413-889e-95bc63b9696c.png"> | <img width="1264" alt="staging-sites-general-settings-after" src="https://user-images.githubusercontent.com/779993/223762345-4c21453d-2108-4923-8ae8-e84d64c05bea.png"> |

## Pre-merge Checklist
